src/ts/stringutils.ts

```diff
@@ -106,12 +106,12 @@ export namespace StringUtils {
         if (skipOffset) {
           time = Math.ceil(skipOffset - player.getCurrentTime());
         } else {
-          time = player.getDuration() - player.getCurrentTime();
+          time = Math.round(player.getDuration() - player.getCurrentTime());
         }
       } else if (formatString.indexOf('playedTime') > -1) {
-        time = player.getCurrentTime();
+        time = Math.round(player.getCurrentTime());
       } else if (formatString.indexOf('adDuration') > -1) {
-        time = player.getDuration();
+        time = Math.round(player.getDuration());
       }
       return formatNumber(time, formatString);
```

Comment:
Nit: You could do a single `Math.round` call here instead and save a few bytes.
```suggestion
      return formatNumber(Math.round(time), formatString);
```

Reply:
👍